README.md

```diff
@@ -9,20 +9,17 @@
   </a>
 </div>
 
-[Temporal](https://temporal.io) is a microservice orchestration platform that enables developers to build scalable applications without sacrificing productivity or reliability. Temporal Server executes units of application logic—Workflows—in a resilient manner that automatically handles intermittent failures and retries failed operations.
+[Temporal](https://temporal.io) is a distributed, scalable, durable, and highly available orchestration engine used to execute asynchronous, long-running business logic in a scalable and resilient way.
```

Comment:
```suggestion
[Temporal](https://temporal.io) is a durable execution system that transparently makes your code durable, fault-tolerant, and simple.
```

Reply:
candace + oxford comma

Reply:
I was going to update after my discussion on Slack with Candace.
Thanks.